worker_generic/manager/cryptography.py

```diff
@@ -0,0 +1,21 @@
+import gnupg
+import base64
+import json
+
+gpg = gnupg.GPG()
+
+'''
+Decrypts a json string
+The json object keys are plain text, the value is gpg encrypted and base64 encoded
+e.g. {"key1": "base64encodedgpgencryptedvalue1", "key2": "base64encodedgpgencryptedvalue2"}
+the method supports only one level of nesting, deeper nesting is not supported yet
+'''
+def decrypt_json(json_encrypted, key):
```

Comment:
Nitpick: it's usually more pythonic to put the docstring of a function below the function header. 😜